hive/src/test/java/com/netease/arctic/hive/io/TestHiveTaskReader.java

```diff
@@ -74,7 +75,10 @@ public void testMergeOnReadFilterTimestampType() {
     Expression filter = Expressions.equal("op_time", "2022-01-01T12:00:00");
     List<KeyedTableScanTask> readTasks = planReadTask(filter);
     // Fields of type INT96 do not save their min/max state, so filter push down is not possible.
-    Assert.assertEquals(5, readTasks.size());
+    PartitionData partitionData = new PartitionData(getArcticTable().spec().partitionType());
```

Comment:
You may want to remove this override implementation and reuse the super implementation as you have solved this stat problem of INT96.